rust_dev_preview/examples/aurora/src/bin/hello-world.rs

```diff
@@ -0,0 +1,44 @@
+use std::fmt::Display;
+
+use aws_sdk_rds::Client;
```

Comment:
Typically we include usings, imports, etc., in the snippet for a HelloWorld so that it is fully runnable as a cut 'n' paste.

Reply:
Done. Using the entire file.